src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.0.0
```

Comment:
are you relying on any semantic conventions from otel? if yes, we'll need to hold off from 1.0.0 stable. If not, we need to document the attribute/tag names in the readme, before stable release.

Reply:
@fred2u ^

Reply:
@cijothomas No, I'm not